src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -180,17 +185,15 @@ class ArchiveExplorer extends React.Component {
             }
 
             if (typeof aItem === 'number' && typeof bItem === 'number') {
-                return aItem - bItem;
+                return sortDirection === SortDirection.ASC ? aItem - bItem : bItem - aItem;
```

Comment:
How about flipping them at assignment to avoid the repeated logic?

```
const aItem = sortDirection === SortDirection.ASC ? a : b;
const bItem = sortdirection === SortDirection.ASC ? b : a;
```

Reply:
> How about flipping them at assignment to avoid the repeated logic?
> 
> ```
> const aItem = sortDirection === SortDirection.ASC ? a : b;
> const bItem = sortdirection === SortDirection.ASC ? b : a;
> ```

Cannot flip them before `null` check, right?